subsys/nrf_security/Kconfig.tls

```
@@ -256,6 +259,14 @@ config MBEDTLS_SSL_KEEP_PEER_CERTIFICATE
 	  reduces RAM usage.
 	  Corresponds to MBEDTLS_SSL_KEEP_PEER_CERTIFICATE in mbed TLS config file.
 
+config MBEDTLS_SSL_TLS1_3_COMPATIBILITY_MODE
+	bool
+	prompt "Enable TLS1.3 middlebox compatibility mode"
```

```suggestion
	bool "Enable TLS1.3 middlebox compatibility mode"
```
(functionally identical)